File: genealogy-ssr/Server/Models/Dtos/Filters/BusinessObjectFilter.cs
using System;

namespace Genealogy.Models
{
    public class BusinessObjectFilter : PaginatorInDto
    {
        public Guid? Id { get; set; }
        public string Name { get; set; }
        public Guid? MetatypeId { get; set; }
        public bool? IsRemoved { get; set; }
        public Guid? UserId { get; set; }
    }
}

File: genealogy-ssr/ClientApp/src/app/models/filters/business-object.filter.ts
import { PaginatorOutDto } from "../dtos";

export interface BusinessObjectFilter extends PaginatorOutDto {
  id?: string;
  name?: string;
  title?: string;
  metatypeId?: string;
  startDate?: Date;
  finishDate?: Date;
  isRemoved?: boolean;
  data?: string;
  userId?: string;
}
